horace_core/lattice_functions/bragg_positions.m

```diff
@@ -171,6 +170,7 @@
         bin_t=trans_bin_width;
         thick_t=trans_thickness;
     else
+        modQ=norm(proj.transform_hkl_to_pix(Qrlu(:)));   % length of Q vector in Ang^-1        
```

Comment:
need for clarification rather than flagging an issue. The move of old line 142 to 173 could be due to (a) calculate modQ only for efficiency reasons; do not evaluate if absolute binning OR (b) ensure that proj here picks up the value set on line 157 rather than pick up the value from line 131 for i==1 OR (c) both. 

Could we delete line 131, use img.proj rather than proj in lines 132-133, and fix the comments at 145-146 so that u and v are appropriately described. Also add comment into the PR description saying what has been fixed

Reply:
Despite the changes I did are probably more correct than the previous one(picking up homoheneous ) , I better restore previous behaviour as do not have sufficient experience in using relative units.